client/src/components/History/Modals/CopyModal.vue

```diff
@@ -72,8 +72,11 @@ export default {
         saveVariant() {
             return this.loading ? "info" : this.formValid ? "primary" : "secondary";
         },
+        userOwnsHistory() {
+            return this.currentUser.id == this.history.user_id;
+        },
         newNameValid() {
-            if (this.name == this.history.name) {
+            if (this.userOwnsHistory && this.name == this.history.name) {
                 return null;
```

Comment:
I know it was like this, but feels more type-consistent to return a boolean here :)
```suggestion
                return false;
```

Reply:
Done ✅ , thanks!
Wondering if this should target `23.1`?

Reply:
Yes, this is fine to target to 23.1 as a bugfix -- that's where it was reported, anyway ;)